x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_overview_logic.test.ts

```diff
@@ -29,10 +36,26 @@ const DEFAULT_CRAWL_RULE: CrawlRule = {
   pattern: '.*',
 };
 
+const MOCK_SERVER_DATA: CrawlerDataFromServer = {
+  domains: [
+    {
+      id: '507f1f77bcf86cd799439011',
+      name: 'moviedatabase.com',
```

Comment:
Should we make this either more generic or Elastic-branded to avoid potential legal/copyright issues? I might be overthinking this though 😅 
```suggestion
      name: 'elastic.co',
```

Reply:
I've fixed this in d551595da4c

Reply:
Looks like this was only fixed on line 76 but not this line (43) - which, speaking of, should we update 76 to DRY out this mock data?

```js
      const crawlerData = {
        domains: [{ ...MOCK_SERVER_DATA[0], defaultCrawlRule: DEFAULT_CRAWL_RULE }],
      };
```

Reply:
Going to go ahead and commit the above suggestion to get this line fixed/the PR approved - DRY comment above is totally optional, feel free to commit in a later PR if you like it (or totally ignore it!)